plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -172,6 +172,11 @@ describe('PersonState.update()', () => {
                 id: expect.any(Number),
                 uuid: uuid.toString(),
                 properties: { a: 1, b: 3, c: 4 },
+                properties_last_operation: {
```

Comment:
Can we add a separate describe block for these (plus timestamp logic) and remove this from the existing tests? 

Ideally a test should be only testing one behavior at a time.

Reply:
I disagree with separating them as the output for the test will give us the information plus seeing this would help debug why the test failed. Otherwise I'd be duplicating the existing tests and the only difference is adding last_operation and timestamp.